include/eve/module/core/regular/binarize.hpp

```diff
@@ -53,7 +53,8 @@ namespace eve
 //!  @godbolt{doc/core/regular/binarize.cpp}
 //! @}
 //================================================================================================
-EVE_MAKE_CALLABLE(binarize_, binarize);
+
```

Comment:
`binarize_not` has its masking de-activated. Why not `binarize` ?

Reply:
done